lib/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php

```
@@ -69,24 +69,27 @@ public function register_routes() {
 	 * @param WP_REST_Request $request Full details about the request.
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
-	public function get_font_collection( $request ) {
+	public function get_item( $request ) {
 		$slug       = $request->get_param( 'slug' );
 		$collection = WP_Font_Library::get_font_collection( $slug );
+
 		// If the collection doesn't exist returns a 404.
 		if ( is_wp_error( $collection ) ) {
 			$collection->add_data( array( 'status' => 404 ) );
-			return $collection;
+			return rest_ensure_response( $collection );
 		}
-		$config_and_data = $collection->get_config_and_data();
-		$collection_data = $config_and_data['data'];
+
+		$config   = $collection->get_config();
+		$contents = $collection->get_content();
 
 		// If there was an error getting the collection data, return the error.
-		if ( is_wp_error( $collection_data ) ) {
-			$collection_data->add_data( array( 'status' => 500 ) );
-			return $collection_data;
+		if ( is_wp_error( $contents ) ) {
+			$contents->add_data( array( 'status' => 500 ) );
+			return rest_ensure_response( $contents );
```

Same as above, I think we can return the error directly.